examples/src/bin/server_acceptor.rs

```diff
@@ -225,8 +225,6 @@ impl TestPki {
         // based on the ClientHello (e.g. selecting a different certificate, or customizing
         // supported algorithms/protocol versions).
         let mut server_config = ServerConfig::builder()
-            .with_safe_default_protocol_versions()
-            .unwrap()
```

Comment:
non-actionable thinking aloud:

with this commit, i probably would have gone for a different route: making `with_safe_default_protocol_versions()` infallible (it returns `WantsVerifier`), but that bakes-in the current property of that function that it enables both protocol versions (and, therefore, there's no route to achieving the degenerate TLS1.2-version but TLS1.3-suites or TLS1.3-version but TLS1.2-suites combinations that are unusable)